examples/scheduled-transaction-multi-sig-threshold.js

```diff
@@ -0,0 +1,109 @@
+const {
+  Client,
+  PrivateKey,
+  KeyList,
+  AccountCreateTransaction,
+  Hbar,
+  AccountBalanceQuery,
+  TransferTransaction,
+  ScheduleSignTransaction,
+  ScheduleInfoQuery,
+  TransactionRecordQuery,
+} = require("@hashgraph/sdk");
+require("dotenv").config();
+
+async function main() {
+  // set up operator
```

Comment:
Can you copy and paste the client setup from other examples. We want the examples to be able to be ran with _either_ `OPERATOR_KEY`, `OPERATOR_ID`, `HEDERA_NETWORK` environment variables, a `CONFIG_FILE` environment variable, or a mixture of both.